test/lit.cfg.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from lit import formats
+
+ROOT_PATH = os.path.realpath(os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
+
+config.name = 'llvm-backend'
+config.test_source_root = os.path.join(ROOT_PATH, "test")
+
+config.test_format = formats.ShTest(True)
+
+config.suffixes = ['.kore']
+
+config.excludes = []
+
+if os.getenv('LIT_USE_NIX'):
```

Comment:
Can we get a comment here about what this is doing?

Reply:
Will comment it - the problem is basically `lit` not inheriting the environment when it's launched, which of course breaks nix derivations.